server/routes/makeAReferral/risk-information/oasys/oasysRiskSummaryView.test.ts

```diff
@@ -6,10 +6,11 @@ import { Risk } from '../../../../models/assessRisksAndNeeds/riskSummary'
 describe('OasysRiskSummaryView', () => {
   describe('oasysRiskInformationArgs', () => {
     describe('additionalRiskInformation', () => {
-      it('returns special content to display if no additional risk information has been provided', () => {
+      it('always returns null text and undefined label', () => {
```

Comment:
could this include why this is preferred?

Reply:
Thanks I've adjusted the tests in this PR to be more descriptive.